src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -98,6 +109,33 @@ internal static class ParameterHelpers
             return parameters;
         }
 
+        private static bool IsPropertyWithoutBody(Symbol owner)
+        {
+            if (owner is SourcePropertySymbol)
+            {
+                var property = (SourcePropertySymbol)owner;
+                var getterWithoutBody = property.GetMethod == null || IsAnyMethodWithoutBody(property.GetMethod);
+                var setterWithoutBody = property.SetMethod == null || IsAnyMethodWithoutBody(property.SetMethod);
+                return getterWithoutBody && setterWithoutBody;
+            }
+
+            return false;
+        }
+
+        private static bool IsAnyMethodWithoutBody(Symbol owner)
+        {
+            if (owner is SourceMethodSymbol)
+            {
+                var method = (SourceMethodSymbol)owner;
+                if (method.BodySyntax == null)
```

Comment:
Possibly... though it seems much more ugly.  We'd have to walk up until we hit a constructor/delegate/method/operator/indexer and would then have to handle them all separately since they all have different syntactic forms.